### PR TITLE
Adding positron and quick edits

### LINKED ISSUE #2453: Doc: Add Positron to intro
Description for the issue.

Adding Positron to intro since Publisher is installed by default in Positron.

### Related Issues

-

```[tasklist]
### Acceptance Criteria
- [ ] Add task or issue references here
```

### PR DESCRIPTION
**Pull request by AshleyHenry15**
## Intent

Resolves #2453 

## Type of Change

- - [ ] Bug Fix 
- - [ ] New Feature 
- - [ ] Breaking Change 
- - [X] Documentation 
- - [ ] Refactor 
- - [ ] Tooling 

## Approach

Positron bundles/installs Publisher by default. I figured that we should add it to these docs since we have it in the Connect guide.

## Checklist

- [ ] I have updated [CHANGELOG.md](../CHANGELOG.md) to cover notable changes.

### CODE REVIEW

**Review by sagerb — APPROVED** [review]
LGTM

**Comment by AshleyHenry15** [comment]
@sagerb - I can't merge this so if someone from the team can take ownership of it - that would be great. 

Also - at some point we should all chat about where this public doc should live/be hosted.

Additionally, can I have write or edit access to the repo so I can help out with doc changes without forking?

**Pull request merged**
